packages/astro/src/@types/astro.ts

```diff
@@ -2428,6 +2428,11 @@ export type AstroFeatureMap = {
 	 * List of features that orbit around the i18n routing
 	 */
 	i18nDomains?: SupportsKind;
+
+	/**
+	 * The adapter is able to support `astro:env`
```

Comment:
We might want to be more specific. It's not a blocker, however the support is for `getEnv`, right?

Reply:
`envGetter`? Even `env` is fine, as long as we're good at documenting it

Reply:
Will spend some time working on jsdoc 👍